web/app/themes/codecorps/templates/content-single.php

```diff
@@ -13,7 +13,15 @@
 
     <div class="blog-post">
       <?php get_template_part('templates/parts/social-share'); ?>
-
+      
+      <p style="font-style: italic;">
```

Comment:
That would be best, but I'm not seeing any WordPress CSS files that are accessed on the live site in the repository. I can either remove the styling or make use of the theme's style.css in the theme's root. I only see https://blog.codecorps.org/app/themes/codecorps/dist/styles/main-4aee62f95a.css referenced in the source.